tools/ci/ci_tools_unittest.sh

```diff
@@ -15,10 +15,8 @@ else
 fi
 
 if [[ $(./wpt test-jobs --includes wptrunner_unittest; echo $?) -eq 0 ]]; then
-    if [ $TOXENV == "py27" ] || [ $TOXENV == "pypy" ]; then
-        cd tools/wptrunner
-        tox
-    fi
+    cd tools/wptrunner
```

Comment:
Uh, this is wptrunner only. We should be running Py3.6 already? Hmm, how did it get so broken anyway?